### DISCUSSION TITLE
Is there any plan to support GPU acceleration?

### DISCUSSION THREAD

**Discussion by wuhongsheng** [ggml-org/whisper.cpp#227] [Q&A]
Is there any plan to support GPU acceleration?

**Comment by misutoneko** [comment]
The [FAQ](https://github.com/ggerganov/whisper.cpp/discussions/126) says no.
So it's really not the focus of the author (and I can very well understand why) but it may be doable.
There are some drop-in replacements for OpenBLAS that could help here (see the issues section).
And you know there are 185 forks already, maybe, just maybe some of them are working towards GPU support?
One can always hope :D
